cli/ssh_test.go

```diff
@@ -407,6 +407,50 @@ func TestSSH(t *testing.T) {
 		pty.WriteLine("exit")
 		<-cmdDone
 	})
+
+	t.Run("FileLogging", func(t *testing.T) {
+		t.Parallel()
+
+		dir := t.TempDir()
+
+		client, workspace, agentToken := setupWorkspaceForAgent(t, nil)
+		inv, root := clitest.New(t, "ssh", workspace.Name, "-l", "--log-dir", dir)
+		clitest.SetupConfig(t, client, root)
+		pty := ptytest.New(t).Attach(inv)
+
+		ctx, cancel := context.WithTimeout(context.Background(), testutil.WaitLong)
```

Comment:
nit: I suppose that `testutil.WaitLong` is on purpose as the test case setups an agent?

Reply:
test case is copied from `ImmediateExit` other than the inclusion of the logging flags.